Editor/Scripts/Server/EditorServer.cs

```diff
@@ -1,31 +1,58 @@
 using System;
-using System.Threading;
 using UnityEditor;
 using UnityEngine;
 using Object = UnityEngine.Object;
 
 namespace Unity.MeshSync.Editor {
 
 [InitializeOnLoad]
-internal static class AutoSetup {
+internal static class EditorServer {
 
-    private static Server m_server;
+    private static Server            m_server;
+
+    private const string CONFIGURATION_TIP =
+        "You can configure the editor server in Edit/ProjectSettings/MeshSync/EditorServer";
```

Comment:
Should be via ProjectSettings, no ?

Reply:
To confirm: "You can configure the editor server via Edit/ProjectSettings/MeshSync/EditorServer"
or just ""You can configure the editor server via ProjectSettings" ?